### DISCUSSION TITLE
Finding craft bakeries

### DISCUSSION THREAD

**Discussion by xeruf** [osmandapp/OsmAnd#12703] [Q&A]
Is there any way to filter for craft bakeries - I would have thought craft=bakery but that seems to be wrong according to [osm wiki](https://wiki.openstreetmap.org/wiki/Tag:craft%3Dbakery) so I guess `shop=bakery` with `bakehouse=yes`, but I don't see that possibility in OsmAnd.

Also, this wouldn't find branches of a craft bakery...

**Comment by edaloze** [comment]
Hi, a French friend of mine created this.
Hope it will help you
http://umap.openstreetmap.fr/fr/map/fours-banaux-boulangeries-et-paysans-boulangers-au_657881#6/47.436/10.020

**Reply by xeruf** [reply]
Hm looks interesting, but I don't know french :/

**Reply by thompehw** [reply]
If you can work in a web browser rather than OsmAnd, then it's very useful to learn how to use Overpass Turbo to interrogate the OSM database. Mapping conventions vary, but by [this query](https://overpass-turbo.eu/s/1bpO) there are only three shops in England tagged shop=bakery and bakehouse=yes

**Comment by edaloze** [comment]
doesn't matter, it's just to show you how it works
